patchwork-events-lifecycle/src/main/java/net/patchworkmc/impl/event/lifecycle/LifecycleEvents.java

```diff
@@ -57,6 +58,15 @@ public static void fireClientTickEvent(TickEvent.Phase phase) {
 		MinecraftForge.EVENT_BUS.post(new TickEvent.ClientTickEvent(phase));
 	}
 
+	public static void fireRenderTickEvent(TickEvent.Phase phase, float renderTickTime) {
```

Comment:
Are either of these methods present in ForgeHooks/ForgeEventFactory? or BasicEventHooks or something? If they are, you should add them to the relevant class in patchwork.

Reply:
Ah yes, they're in BasicEventHooks.